examples/kubernetes/multicluster-service.yaml

```diff
@@ -1,9 +1,29 @@
+apiVersion: config.gateway.envoyproxy.io/v1alpha1
+kind: EnvoyProxy
+metadata:
+  name: custom-proxy-config
+  namespace: envoy-gateway-system
+spec:
+  logging:
+    level:
+      default: warn
```

Comment:
Is the logging level setting necessary?

Reply:
Not necessary but does not impact anything, `default/warn` is the default value of logging.

Reply:
Can we remove it since the default logging level is  already `warn`.

Reply:
Done